.travis.yml

```diff
@@ -10,8 +10,8 @@ osx: &osx
    language: generic
 matrix:
    include:
-      - <<: *linux
-        env: CONAN_GCC_VERSIONS=4.9 CONAN_DOCKER_IMAGE=lasote/conangcc49
+#      - <<: *linux
```

Comment:
Is there some error building on gcc 4.9 ?

Reply:
It has a linking problem with the m library when cross-compiling (x86 build on a 64 bit vm) that I just couldn't resolve: https://travis-ci.org/dimi309/conan-vorbis/jobs/361324985 In general, the gcc 4.9 image has recently started producing crashes of this sort with my other libraries too...

Reply:
you need to exclude the condition `and not self.options.shared` at line 69

Reply:
I've already tried that :) 

https://github.com/dimi309/conan-vorbis/commit/13e59d79bdbb2ae1f542194cc9fd449f03467d0c

It didn't work. In general, when linking dynamically with a library you don't need to link to such dependencies because the .so file has all it needs as far as I know. But I still did give it a try.

Reply:
Thanks to share.
I'm just afraid if in the future we use vorbis as dependency for some package. So we need to remove gcc 4.9 for any package.
I'll take a look later if I can discover some workaround to fix.

Reply:
It will be good if it gets fixed, but note that it's not that the package does not work with gcc 4.9. It does not work with gcc 4.9 compiling the 32 bit version on a travis virtual machine using the specific image. I have never had trouble compiling it locally and it even used to work on this particular image, which has been giving me trouble with a lot of packages lately. I'm not sure what has happened...

Reply:
vorbis is dependency for sure, e.g. for ffmpeg, so I'll take care to get it fixed